recipes/cpc2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cpc2" %}
+{% set version = "1.0.1" %}
+{% set file_ext = "tar.gz" %}
+{% set sha256 = "31f3d1e2466aea25f03ffcb5abd4556cd17862ef65ed3a1e42f3d99665529d08" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/gao-lab/CPC2_standalone/archive/refs/tags/v{{ version }}.{{ file_ext }}
+  sha256: {{ sha256 }}
+  path: ..
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - setuptools
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - setuptools 
+    - pip
+    - python
+    - biopython
+    - six
```

Comment:
Do you need everything here in host? Biopython and six at least are probably only runtime deps. See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host.

Reply:
They are needed in the `run:` section, but probably not the `host:` section